Marlin/Marlin_main.cpp

```diff
@@ -1579,7 +1647,7 @@ static void setup_for_endstop_move() {
       feedrate = homing_feedrate[Z_AXIS];
 
       // Move down until the Z probe (or endstop?) is triggered
-      float zPosition = -(Z_MAX_LENGTH + 10);
+      float zPosition = MIN_PROBE_PT;
```

Comment:
> Why exactly do you think the position is lost during probing?

I don't mean to say it's lost, just not accurate or especially meaningful. During probing, the Z position is just a working number for moving up and down, and not necessarily guaranteed to be an accurate reflection of the Z-distance to the bed. At least, that is how I have tended to think of it. But maybe that has changed…?